### PR TITLE
Merge 1.5 into master

### PR DESCRIPTION
**Pull request by skeleton**

### CODE REVIEW

File: features/Context/FixturesContext.php [hunk 1]
```diff
@@ -1692,6 +1685,14 @@ protected function createProductGroup($code, $label, $type, array $attributes, a
     }
 
     /**
+     * @return \Pim\Bundle\CatalogBundle\Factory\GroupFactory
```
Comment (pierallard) [inline features/Context/FixturesContext.php]: Return GroupFactory (use use)

File: features/Context/DataGridContext.php [hunk 2]
```diff
@@ -354,13 +354,24 @@ public function iShowTheFilter($filterName)
...
     {
         $this->datagrid->hideFilter($filterName);
     }
 
     /**
+     * @param string $filterName
+     *
+     * @Then /^I expand the "([^"]*)" sidebar$/
+     */
+    public function iexpandTheCategoriesSidebar($filterName)
```
Comment (pierallard) [inline features/Context/DataGridContext.php]: iExpand

File: features/Context/Page/Base/Grid.php [hunk 3]
```diff
@@ -209,8 +209,33 @@ public function findAction($element, $actionName)
...
+     * @throws \InvalidArgumentException
+     */
+    public function checkOptionInFilter(array $expectedOptions, $filterName)
+    {
+        $filter = $this->getFilter($filterName);
+
+        $filter->open();
+        $options = $filter->getOptions();
+
+        if ($options != $expectedOptions) {
```
Comment (pierallard) [inline features/Context/Page/Base/Grid.php]: !== ? Not sure.

**Comment by pierallard** [comment]
Little comments then GTM

File: features/Context/Page/Base/Grid.php (hunk 3 above)
Comment (skeleton) [inline features/Context/Page/Base/Grid.php]: with the operator `===`, when you compare 2 array, order is checked and can return false if thse 2 array are not exactly equals
with `==` data are well checked, but doesn't care about order

**Comment by damien-carcel** [comment]
GTM

**Pull request merged**